LICENSE-binary

```diff
@@ -256,7 +257,6 @@ org.apache.hadoop:hadoop-mapreduce-client-jobclient
 org.apache.hadoop:hadoop-mapreduce-client-shuffle
 org.apache.hadoop:hadoop-yarn-api
```

Comment:
I think the diff of this patch makes sense, but for existing items, are those hadoop libs included in our binary tarball, except for `hadoop-client-api` and `hadoop-client-runtime`?

Reply:
I think other Hadoop libs can be removed.